tools/clang/test/CodeGenSPIRV/switch-stmt.opswitch.hlsl

```diff
@@ -0,0 +1,319 @@
+// CHECK-NEXT: OpBranch %switch_merge_2
+    default:
+      result = 777;
+      break;
+  }
+// CHECK-NEXT: %switch_merge_2 = OpLabel
+
+
+
+  ///////////////////////////////////////
+  // Fall-through cases with no body   //
+  ///////////////////////////////////////
+
+// CHECK-NEXT: [[a4:%\d+]] = OpLoad %int %a
+// CHECK-NEXT: OpSelectionMerge %switch_merge_3 None
+// CHECK-NEXT: OpSwitch [[a4]] %switch_default_1 10 %switch_10 11 %switch_11 12 %switch_12
+  switch(a) {
+// CHECK-NEXT: %switch_10 = OpLabel
+// CHECK-NEXT: OpBranch %switch_11
+    case 10:
+// CHECK-NEXT: %switch_11 = OpLabel
+// CHECK-NEXT: OpBranch %switch_default_1
+    case 11:
+// CHECK-NEXT: %switch_default_1 = OpLabel
+// CHECK-NEXT: OpBranch %switch_12
+    default:
+// CHECK-NEXT: %switch_12 = OpLabel
+// CHECK-NEXT: OpStore %result %int_12
+// CHECK-NEXT: OpBranch %switch_merge_3
+    case 12:
+      result = 12;
+  }
+// CHECK-NEXT: %switch_merge_3 = OpLabel
+
+
+
+  ////////////////////////////////////////////////
+  // No-op. Two nested cases and a nested break //
+  ////////////////////////////////////////////////
+
+// CHECK-NEXT: [[a5:%\d+]] = OpLoad %int %a
+// CHECK-NEXT: OpSelectionMerge %switch_merge_4 None
+// CHECK-NEXT: OpSwitch [[a5]] %switch_merge_4 15 %switch_15 16 %switch_16
+  switch(a) {
+// CHECK-NEXT: %switch_15 = OpLabel
+// CHECK-NEXT: OpBranch %switch_16
+    case 15:
+// CHECK-NEXT: %switch_16 = OpLabel
+// CHECK-NEXT: OpBranch %switch_merge_4
+    case 16:
+      break;
+  }
+// CHECK-NEXT: %switch_merge_4 = OpLabel
+
+
+
+  ////////////////////////////////////////////////////////////////
+  // Using braces (compound statements) in various parts        //
+  // Using breaks such that each AST configuration is different //
+  // Also uses 'forcecase' attribute                            //
+  ////////////////////////////////////////////////////////////////
+
+// CHECK-NEXT: [[a6:%\d+]] = OpLoad %int %a
+// CHECK-NEXT: OpSelectionMerge %switch_merge_5 None
+// CHECK-NEXT: OpSwitch [[a6]] %switch_merge_5 20 %switch_20 21 %switch_21 22 %switch_22 23 %switch_23 24 %switch_24 25 %switch_25 26 %switch_26 27 %switch_27 28 %switch_28 29 %switch_29
+  [forcecase] switch(a) {
+// CHECK-NEXT: %switch_20 = OpLabel
+// CHECK-NEXT: OpStore %result %int_20
+// CHECK-NEXT: OpBranch %switch_merge_5
+    case 20: {
+      result = 20;
+      break;
+    }
+// CHECK-NEXT: %switch_21 = OpLabel
+// CHECK-NEXT: OpStore %result %int_21
+// CHECK-NEXT: OpBranch %switch_merge_5
+    case 21:
+      result = 21;
+      break;
+// CHECK-NEXT: %switch_22 = OpLabel
+// CHECK-NEXT: OpBranch %switch_23
+// CHECK-NEXT: %switch_23 = OpLabel
+// CHECK-NEXT: OpBranch %switch_merge_5
+    case 22:
+    case 23:
+      break;
+// CHECK-NEXT: %switch_24 = OpLabel
+// CHECK-NEXT: OpBranch %switch_25
+// CHECK-NEXT: %switch_25 = OpLabel
+// CHECK-NEXT: OpStore %result %int_25
+// CHECK-NEXT: OpBranch %switch_merge_5
+    case 24:
+    case 25:
+      result = 25;
```

Comment:
This pattern is already tested in the above cases. What about changing it into `case 25: { result = 25; } break;`?